lib/features/hooks/hooks.rb

```diff
@@ -60,7 +76,6 @@
       $logger.info "BrowserStack session(s): #{url}"
     end
   end
-  MazeRunner.driver.start_driver unless config.appium_session_isolation
 
   # Call any blocks registered by the client
   MazeRunner.hooks.call_after_configuration config
```

Comment:
This isn't related to the PR but I think the `next if config.farm == :none` on line 13 will mean these `after_configuration` never get called

Reply:
PLAT-5666 raised for this.